public/javascripts/full_system.js

```diff
@@ -64,9 +64,12 @@ function UpdateOnce() {
       if(data['status'] == 'offline')
         $(`#${r}_status`).html('OFFLINE');
       else {
-        $.getJSON(`/devices/get_last_point?sensor=${r}`, (val) => {
+        let roi = data['readout_interval'];
+        $.getJSON(`/devices/get_last_point?sensor=${r}&roi=${roi}`, (val) => {
```

Comment:
We should still show old readings (at least much older than a readout_interval, even if month-old is not interesting). How about identifying them more prominently e.g. with ⚠️ instead?